app/controllers/admin/venue_commercials_controller.rb

```diff
@@ -0,0 +1,62 @@
+module Admin
+  class VenueCommercialsController < Admin::BaseController
+    load_and_authorize_resource :conference, find_by: :short_title
+    load_and_authorize_resource :venue, through: :conference, singelton: true
```

Comment:
Yeah, that would also work. Instead of this I was using @conference.venue, but ratherless, I guess this is better idea.